cmd/maya-apiserver/app/server/backup_endpoint_v1alpha1.go

```diff
@@ -80,8 +80,8 @@ func (bOps *backupAPIOps) create() (interface{}, error) {
 		return nil, CodedError(400, fmt.Sprintf("Failed to create backup '%v': missing volume name", bkp.Spec.BackupName))
 	}
 
-	// backupIP is expected
-	if len(strings.TrimSpace(bkp.Spec.BackupDest)) == 0 {
+	// backupIP is expected for cloud snapshot
```

Comment:
n_: cloud -> remote ( as minio can also be used)

Reply:
ok